compose-builder/add-mqtt-messagebus.yml

```diff
@@ -22,6 +22,7 @@ services:
     environment:
       MESSAGEQUEUE_OPTIONAL_CLIENTID: core-data
       MESSAGEQUEUE_AUTHMODE: none
+      MESSAGEQUEUE_HOST: edgex-mqtt-broker
```

Comment:
Change not needed. Override should stay in mqtt-bus.env since these are the only services that use it.

Reply:
@lenny-intel If the both file didn't change, the base service of `MESSAGEQUEUE_HOST` will be updated to `edgex-redis` on mqtt bus compose files. Please see the [compose file](https://github.com/cherrycl/edgex-compose/blob/a394907a31d4d87ff3e5be5a073442d259ce5481/taf/docker-compose-taf-mqtt-bus.yml#L584) on my test branch `issue-312-test` only update common.env and docker-compose-base.yml.

Reply:
Humm, ok so the values in the .env files are not overriding the existing values in the environment section as we expected them to.  Bummer.